tests/telemetry/test_telemetry.py

```diff
@@ -19,14 +21,28 @@ def get_list_stdout(cmd_out):
         out_list.append(result)
     return out_list
 
+def restart_telemetry(duthost):
```

Comment:
this is about feature enable/disable? why call it restart telemtry?

Reply:
if you plan to do restart, then just do duthost.server(state=restarted).

Reply:
This function doesn't appear to restart the telemetry service, but rather it simply enables it if it is disabled. I think the function is poorly named.

Reply:
renamed function and updated duthost.service(state=restarted).